components/mailer.rst

```diff
@@ -177,6 +177,43 @@ the application or when using a self-signed certificate::
 
     The ``verify_peer`` option was introduced in Symfony 5.1.
 
+Other Options
+-------------
+
+.. versionadded:: 5.2
+
+    These options were added in Symfony 5.2.
+
+command
+~~~~~~~
+
+Command to be executed by ``sendmail`` transport.
+
+    $dsn = 'sendmail://default?command=/usr/sbin/sendmail%20-oi%20-t'
+
+local_domain
+~~~~~~~~~~~~
+
+The domain name to use in ``HELO`` command.
+
+    $dsn = 'smtps://smtp.example.com?local_domain=example.org'
+
+restart_threshold
+~~~~~~~~~~~~~~~~~
+
+The maximum number of messages to send before re-starting the transport. Can be 
+used together with ``restart_threshold_sleep`` which is the number of seconds to 
```

Comment:
This option is not documented, isn't it? 🤔

Reply:
Indeed, I'll ammend it with the other changes.